### PR TITLE
SHS-5113: Editoria11y Customizations

### PR DESCRIPTION
**Pull request by cienvaras**
# READY FOR REVIEW

## Summary
- Change Editoria11y tab title: "Outline" → "Heading Outline"
- Change styles of alert button to make text black

## Need Review By (Date)
11/27

## Urgency
medium

## Steps to Test
1. Visit any page as a "Developer" user. You should see the Editoria11y widget in the bottom right corner
2. Click on the widget. You should see a panel with three tabs. The second tab should have the title "Heading Outline" instead of "Outline"
    Screenshot 2023-11-23 at 13 21 41
4. Visit a page with an Editoria11y alert status (red outline widget). If you can't find one, you can create a flexible page and add an empty heading in the Text Area. Confirm that the widget error count text is black instead of red.
    Screenshot 2023-11-23 at 13 25 19

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)

### CODE REVIEW

File: docroot/themes/humsci/humsci_basic/src/js/shared/editoria11y/editoria11y.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  // Editoria11y element is added to the DOM after the load event, we need to
+  // observe the body element for changes.
+  const observer = new MutationObserver((mutationList) => {
+    mutationList.forEach((mutation) => {
+      if (mutation.addedNodes.length && mutation.addedNodes[0].nodeName === 'ED11Y-ELEMENT-PANEL') {
+        // Once we get the element, we update the styles to make the alert button text black.
+        const style = mutation.addedNodes[0].shadowRoot.querySelector('style');
+        style.textContent = `${style.textContent}
+        .shut.errors #toggle {
+          color: #000;
```
Comment (mndonx) [inline docroot/themes/humsci/humsci_basic/src/js/shared/editoria11y/editoria11y.js]: Minor, but I think in general non-abbreviated hexadecimal values are preferred.

Comment (mndonx) [inline docroot/themes/humsci/humsci_basic/src/js/shared/editoria11y/editoria11y.js]:
```suggestion
          color: #000000;
```

**Review by mndonx — CHANGES_REQUESTED** [review]
One minor change. I tested and it works as described in the PR.

File: docroot/themes/humsci/humsci_basic/src/js/shared/editoria11y/editoria11y.js (hunk 1 above)
Comment (cienvaras) [reply docroot/themes/humsci/humsci_basic/src/js/shared/editoria11y/editoria11y.js]: @mndonx Thanks, fixed!

**Pull request merged**